README.md

```diff
@@ -1,16 +1,28 @@
 # StaffConnect
 
-[![CI Status](https://github.com/se-edu/addressbook-level3/workflows/Java%20CI/badge.svg)](https://github.com/se-edu/addressbook-level3/actions)
+[![CI Status](https://github.com/AY2324S2-CS2103-F08-3/tp/actions/workflows/gradle.yml/badge.svg)](https://github.com/AY2324S2-CS2103-F08-3/tp/actions)
 
 ![Ui](docs/images/Ui.png)
 
-* This is **a sample project for Software Engineering (SE) students**.<br>
-  Example usages:
-  * as a starting point of a course project (as opposed to writing everything from scratch)
-  * as a case study
-* The project simulates an ongoing software project for a desktop application (called _AddressBook_) used for managing contact details.
-  * It is **written in OOP fashion**. It provides a **reasonably well-written** code base **bigger** (around 6 KLoC) than what students usually write in beginner-level SE modules, without being overwhelmingly big.
-  * It comes with a **reasonable level of user and developer documentation**.
-* It is named `AddressBook Level 3` (`AB3` for short) because it was initially created as a part of a series of `AddressBook` projects (`Level 1`, `Level 2`, `Level 3` ...).
-* For the detailed documentation of this project, see the **[Address Book Product Website](https://se-education.org/addressbook-level3)**.
-* This project is a **part of the se-education.org** initiative. If you would like to contribute code to this project, see [se-education.org](https://se-education.org#https://se-education.org/#contributing) for more info.
+## StaffConnects offers a quick and easy way to store and review contacts of professors/tutors.<br>
+
+StaffConnect offers convenience and peace of mind to a struggling student with memory recollection issues.
+This product allows users to easily identify and connect with educators by providing visual cues alongside contact
+details, supported by a quick and easy filter system. While it has a GUI, most of the user interactions happen using a CLI (Command Line Interface).
+
+# Features:
+* Quickly add and review contacts
+* Filter and search your required contacts quickly
+* Add consultation times to the particular contact
+* View and manage your meeting times
+
+# Getting Started
+ * Ensure that at least **Java 11 with JavaFx runtime is installed**
+ * Run the the application with ```java -jar StaffConnect.jar```
+        
+# Credits
+StaffConnect is an ongoing software project for a desktop application used for managing contact details.
+* It is **written in OOP fashion**. It provides a **reasonably well-written** code base **currently** (around 6 KLoC).
+* For the detailed documentation of this project, see the **[StaffConnect Product Website](https://ay2324s2-cs2103-f08-3.github.io/tp/)**.
```

Comment:
I like how you added this portion into the readme. Any reason why you added it under the Credits section instead of after the Ui.png?

Reply:
Has been fix in the new commit, they were leftover lines from the pervious template